### ISSUE TITLE
[Request] Reconfigure with new integration UI

### ISSUE THREAD

**Issue by KairuByte** [And3rsL/Deebot-for-Hassio#103]
The recent change to adding the integration through the UI is handy, but leaves something to be desired when altering a setting. Password changes, live map, room coloring, etc. all require you to delete the integration, restart HA, and re-add the integration.

IMHO this all but defeats the benefits of the UI configuration if anyone needs to alter the settings even once.

**Comment by And3rsL** [comment]
Yes, We are aware of the problem and it will be fixed in future releases 😃 

we hope to introduce more settings via UI

**Comment by robertwigley** [comment]
> The recent change to adding the integration through the UI is handy, but leaves something to be desired when altering a setting. Password changes, live map, room coloring, etc. all require you to delete the integration, restart HA, and re-add the integration.
> 
> IMHO this all but defeats the benefits of the UI configuration if anyone needs to alter the settings even once.

Until this enhancement is made, you can always edit the entries in ..\config\.storage\core.config_entries directly rather than removing, restarting and re-adding the integration. Not ideal, but quicker than removing and re-adding. Note, this can be done for any integration that you can't reconfigure in the front end.
